Software.py

```diff
@@ -256,19 +221,19 @@ def on_modified_async(self, view):
         if fileInfoData is None:
             return
 
-        fileSize = view.size()
+        fileSize = get_character_count(view)
 
-        #lines = 0
+        # lines = 0
         # rowcol gives 0-based line number, need to add one as on editor lines starts from 1
```

Comment:
Good idea to move this logic, might delete the comment above since you already mention it in the method

Reply:
👍